internal/receiver/smartagentreceiver/README.md

```diff
@@ -19,30 +19,58 @@ should be used.
 [Collector processors](https://github.com/open-telemetry/opentelemetry-collector/blob/master/processor/README.md).
 1. Monitors with [dimension property and tag update
 functionality](https://dev.splunk.com/observability/docs/datamodel#Creating-or-updating-custom-properties-and-tags)
-require an associated `dimensionClients` field that references the name of the SignalFx exporter you are using in your
-pipeline.
+allow an associated `dimensionClients` field that references the name of the SignalFx exporter you are using in your
+pipeline.  If you do not specify any exporters via this field, the receiver will attempt to use the associated
+pipeline.  If the next element of the pipeline isn't compatible with dimension update behavior, if a lone SignalFx
+exporter was configured for your deployment, it will be selected.  If no dimension update behavior is desired,
+you can specify the empty array `[]` to disable.
 1. Monitors with [event-sending
-functionality](https://dev.splunk.com/observability/docs/datamodel/ingest#Send-custom-events)
-require an associated `eventClients` field that references the name of the SignalFx exporter you are using in your
-pipeline.
+functionality](https://dev.splunk.com/observability/docs/datamodel/ingest#Send-custom-events) should be made members of
+a `logs` pipeline that utilizes a [SignalFx
+exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/signalfxexporter/README.md)
+that will make the event submission requests.  It's recommended, and in the case of the Processlist monitor required,
+to use a Resource Detection to ensure that host identity and other useful information is made available as event
+dimensions.
 
 Example:
 
 ```yaml
-exporters:
-  signalfx:
+extensions:
+  smartagent:
```

Comment:
Is the extension required even if we don't specify any settings for it?

Reply:
Not necessary unless it changes settings.  Will remove for now.